src/Game.java

```diff
@@ -1,39 +1,24 @@
 public class Game {
-
-    int height = 6;
-    char[][] map = new char[height][height];
-
     boolean victory = false;
     boolean gameOver = false;
-    Hero hero = new Hero(height, map, 'H');
-    Stone stone = new Stone(height, map, 'O');
-    Hole hole = new Hole(height, map, 'X');
-    Trap trap = new Trap(height, map, '8');
-
-
-    public Game() {
-        createFrame();
-        hero.newPosition(map);
-        stone.newPosition(map);
-        hole.newPosition(map);
-        trap.newPosition(map);
-    }
-
-    public void createFrame() {
-        for (int i = 0; i < height; i++) {
-            for (int j = 0; j < height; j++) {
 
-                if (i == 0 || i == height - 1 || j == 0 || j == height - 1)
-                    map[i][j] = '+';
-
-                else
-                    map[i][j] = ' ';
-            }
-        }
+    int height;
+    char[][] map;
+
+    public Game(
```

Comment:
Teraz jak na to patrzę to pewnie dobrze by było gdyby była jakaś klasa, może map, która zawiera te wszystkie rzeczy którą zawiera gra.
Wtedy klasa Game by była trochę mniejsza